src/main/resources/logback.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to Apereo under one or more contributor license
+    agreements. See the NOTICE file distributed with this work
+    for additional information regarding copyright ownership.
+    Apereo licenses this file to you under the Apache License,
+    Version 2.0 (the "License"); you may not use this file
+    except in compliance with the License.  You may obtain a
+    copy of the License at the following location:
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<!-- For assistance related to logback-translator or configuration  -->
+<!-- files in general, please contact the logback user mailing list -->
+<!-- at http://www.qos.ch/mailman/listinfo/logback-user             -->
+<!--                                                                -->
+<!-- For professional support please see                            -->
+<!--    http://www.qos.ch/shop/products/professionalSupport         -->
+<!--                                                                -->
+<configuration scan="true" scanPeriod="30 seconds">
+  <contextName>BookmarksPortlet</contextName>
+
+  <!--
+    | Propagate log levels to java.util.logging
+  +-->
+  <contextListener class="ch.qos.logback.classic.jul.LevelChangePropagator">
+    <resetJUL>true</resetJUL>
+  </contextListener>
+
+  <!--
+    | Expose the logback configuration via JMX
+  +-->
+  <jmxConfigurator />
+
+  <!--
+    | Specify a local property that sets up the logging directory (to tomcat's logs directory).
+  +-->
+  <property scope="local" name="LOG_DIR" value="${bookmarks.root}/../../logs" />
```

Comment:
Should `LOG_DIR` be set/configured like https://github.com/uPortal-Project/NotificationPortlet/blob/master/notification-portlet-webapp/src/main/resources/logback.xml#L48-L53 ?

Reply:
Yeah, let's make it the default to expect the Tomcat layout.